src/shared/install.c

```diff
@@ -116,6 +116,14 @@ bool unit_type_may_template(UnitType type) {
                       UNIT_PATH);
 }
 
+static const char *unitfiletype_table[_UNIT_FILE_TYPE_MAX] = {
+        [UNIT_FILE_TYPE_REGULAR] = "regular",
+        [UNIT_FILE_TYPE_SYMLINK] = "symlink",
+        [UNIT_FILE_TYPE_MASKED] = "masked",
+};
+
+DEFINE_PRIVATE_STRING_TABLE_LOOKUP_TO_STRING(unitfiletype, UnitFileType);
```

Comment:
Not that it really matters: usually when mapping CamelCase syntax into underscore_separated syntax `FooBar` becomes `foo_bar` in our sources. Hence, call the table `unit_file_type_table`?